### PR TITLE
Start of private repro 

### PR DESCRIPTION
**Pull request by andreas-c-hofmann**

### CODE REVIEW

File: achlnk/.htaccess [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+
+redirect 301 /0f891fbc https://www.historicum.net/dhb
```
Comment (davidlehn) [inline achlnk/.htaccess:3]: I think this is the first use of `mod_alias`.  Everyone else is using `mod_rewrite`.  Did you test this?  I think you need the full path when using `redirect`.

File: achlnk/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# private Repository
```
Comment (davidlehn) [inline achlnk/README.md:1]: What does this comment mean?  We prefer to host redirects for public projects than private ones.

In any case, please add a list of maintainer github account ids.

**Comment by andreas-c-hofmann** [comment]
Am 12.01.2023 um 04:10 schrieb David I. Lehn:
> ***@***.**** commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In achlnk/.htaccess
> <https://github.com/perma-id/w3id.org/pull/3067#discussion_r1067657983>:
>
>> @@ -0,0 +1,3 @@
> +Options +FollowSymLinks
> +
> +redirect 301 /0f891fbc https://www.historicum.net/dhb
>
> I think this is the first use of |mod_alias|. Everyone else is using
> |mod_rewrite|. Did you test this? I think you need the full path when
> using |redirect|.

I want to add point to point redirects with virtually created pathnames.
I tested it in a private environment. The virtual path needs to be
relative and the target needs to be absolute.

> ------------------------------------------------------------------------
>
> In achlnk/README.md
> <https://github.com/perma-id/w3id.org/pull/3067#discussion_r1067658818>:
>
>> @@ -0,0 +1 @@
> +# private Repository
>
> What does this comment mean? We prefer to host redirects for public
> projects than private ones.
>
> In any case, please add a list of maintainer github account ids.

I added some explanations to the readme

**Comment by dgarijo** [comment]
@davidlehn is it ok to merge this PR?

**Comment by davidlehn** [comment]
> I want to add point to point redirects with virtually created pathnames.
> I tested it in a private environment. The virtual path needs to be
> relative and the target needs to be absolute.

> @davidlehn is it ok to merge this PR?

@dgarijo @andreas-c-hofmann No, this still won't work.  `mod_alias` docs and a simple test confirm.  `Redirect` matches on the full URL path, not just the local path.  Use `mod_rewrite` if you want relative behavior.  Otherwise prefix with `/achlnk`.

https://httpd.apache.org/docs/2.4/mod/mod_alias.html#redirect

**Comment by dgarijo** [comment]
Ping @andreas-c-hofmann to address @davidlehn's concerns

**Comment by dgarijo** [comment]
Closing due to inactivity. Please reopen if appropriate

**Pull request closed without merging**